playground/mocks/data/api/v1/authn/forced-idp-discovery-linkedin-idp.json

```diff
@@ -0,0 +1,16 @@
+{
+  "subject": "okta:acct:undefined",
```

Comment:
nit:
looks like this subject is not checked anywhere, but to keep it consistent change it to `okta:acct:`.